reactive/puppet.py

```diff
@@ -50,26 +36,19 @@ def install_puppet():
 @when_not('puppet.available')
 def puppet_ready():
     """
-    Set status so users know puppet is ready. Set the 'puppet.available' state
-    so other layers can gate puppet operations.
+    Set the `puppet.available` state so other layers can gate
+    puppet operations.
     """
-    hookenv.status_set('active', 'puppet is installed')
```

Comment:
I'm not sure how much value there is in deploying this by itself.  If that is something valuable, then we still need to handle setting the status better so as to not conflict with charms building off this.